invisible_cities/io/mcinfo_io.py

```diff
@@ -325,19 +325,19 @@ def get_event_numbers_in_file(file_name: str) -> np.ndarray:
 
 
 def _get_list_of_events_new(h5in : tb.file.File) -> np.ndarray:
-    mc_tbls  = ['hits', 'particles', 'sns_response']
-    def try_unique_evt_itr(group, itr):
-        for elem in itr:
-            try:
-                yield np.unique(getattr(group, elem).cols.event_id)
-            except tb.exceptions.NoSuchNodeError:
-                pass
-
-    evt_list = list(try_unique_evt_itr(h5in.root.MC, mc_tbls))
-    if len(evt_list) == 0:
-        raise AttributeError("At least one of MC/hits, MC/particles, \
-        MC/sns_response must be present to use get_list_of_events.")
-    return np.unique(np.concatenate(evt_list)).astype(int)
+
+    def _get_event_ids_table(tablename):
```

Comment:
Leading underscores in function local scopes don't really make much sense.

Reply:
Changed